testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kafka-ssl/secrets/create-certs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
I don't find this file used.

Reply:
Hello @kokosing , 

I'm planning on reusing this file to generate dynamically SSL certs when implementing SASL and removing current generated Kafka certs.

The reasons I find dynamically generated certs better is because:
- committing generated certs can increase the repo size in the long run;
- adding new security protocol e.g. SASL_SSL will require generating new certs;
- committed certs will expire some day even if its set to currently 9999 days (27 years). Someone will have to regenerate all of them which is not a fun thing I would say :).